app/utils/isStringNumber.ts

```diff
@@ -1,3 +1,4 @@
 const isStringNumber = (string: string): boolean => /^\d+$/.test(string);
 
 export default isStringNumber;
+export { isStringNumber };
```

Comment:
why do we need both a default export and a named export here?

Reply:
the compiler has rules that are causing it to fail if the name of the imported value is the same as the default export's name